api_server/src/request/actions.rs

```diff
@@ -46,6 +47,13 @@ fn validate_payload(action_body: &ActionBody) -> Result<(), String> {
                 None => return Err("Payload is required for block device rescan.".to_string()),
             }
         }
+        ActionType::FlushMetrics => {
```

Comment:
It looks like validate_payload for FlushMetrics should have the same behaviour as `InstanceStart`. You can write them as one and make the error message dependent on the ActionType string representation.
Something like:
```rust
...
        ActionType::FlushMetrics | ActionType::InstanceStart => {
            if !action_body.payload.is_none() {
                return Err(format!("{:?} does not support a payload.", action_body.action_type));
            }
            Ok(())
        }
```